.github/workflows/build_ci.yml

```
@@ -42,7 +42,10 @@ jobs:
         with:
           submodules: true
 
-      - name: Build
+      - name: Insrall dependencies
```

```suggestion
      - name: Install dependencies
```
Small typo slipped in. Was pasted to other places as well